amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Can you create a `schema/v0.2` directory and put this file in there? I believe we are going to have several versions of this file, so it is good to start with such a structure.

Reply:
It should refer to the version of this schema. Chose whatever version you think makes sense :)